components/bl-list-component/src/index.js

```diff
@@ -0,0 +1,19 @@
+import { List } from './list';
+
+export default function ListComponent ({
+  component, eventHandlers
+}) {
+
+  const { cn } = BackendlessUI.CSSUtils;
```

Comment:
That should be moved out from the component. You don't need to create cn every render